stable/insights-agent/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+appVersion: 0.0.1
+description: A Helm chart to run the Fairwinds Insights agent
+name: insights-agent
+version: 0.4.0
```

Comment:
Curious why the chart is starting at 0.4.0 and the app at 0.0.1.  I would expect the opposite.

Reply:
This is a weird chart, since it involves several different "apps" (polaris, kube-hunter, goldilocks, ...).

I made `appVersion` `0.0.12` which matches the current version of the Insights application (which, ironically, is _not_ the application installed by this agent. but it's where the data is going, so...)

Reply:
¯\(◉‿◉)/¯